src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/ExprEquipCompModel.java

```diff
@@ -0,0 +1,99 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.NamespacedKey;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+import org.skriptlang.skript.log.runtime.SyntaxRuntimeErrorProducer;
+
+@Name("Equippable Component - Model")
+@Description("The model of the item when equipped. "
+	+ "Note that equippable component elements are experimental making them subject to change and may not work as intended.")
+@Example("set the model key of {_item} to \"custom_model\"")
+@Example("""
+	set {_component} to the equippable component of {_item}
+	set the model id of {_component} to \"custom_model\"
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class ExprEquipCompModel extends PropertyExpression<EquippableWrapper, String> implements EquippableExperiment, SyntaxRuntimeErrorProducer {
+
+	static {
+		register(ExprEquipCompModel.class, String.class, "model (key|id)", "equippablecomponents");
```

Comment:
```suggestion
		register(ExprEquipCompModel.class, String.class, "equipped model (key|id)", "equippablecomponents");
```
again, a bit more specificity.